### PR TITLE
Add config for multi domain usage fixes #122

### LINKED ISSUE #122: Selectable categories with multiple domains
Hello,

I am using the seminars extension for multiple websites in one TYPO3 installation. After creating a seminar on the first website, all categories of the project are shown - also the ones from other websites, which is not what I want.
It would be nicer, if there was one folder, that contained the categories, organizers/hosts and locations and could further be browsed in recursively.
Or is there already a solution for this out there, that I am not seeing?

Thanks a lot in advance!
Cheers,

### PR DESCRIPTION
**Pull request by leofuehringer**
It is possible to set a folder uid in the page TSconfig, which contains the categories, places, etc. Only this elements are shown in flexforms and TCA. It is helpful when a multi domain setup is used.

PS: It is my first experience with git. I hope that everything is alright the way I did it.

### CODE REVIEW

**Comment by oliverklee** [comment]
Hi @leofuehringer, thanks a lot for the PR! <3 I'll take a look at it. It might take some time, though, as I'll be giving a workshop at the start of September and will need the time before it to prepare.

File: Configuration/TCA/tx_seminars_seminars.php [hunk 1]
```diff
@@ -162,12 +161,12 @@
             'exclude' => 1,
             'label' => 'LLL:EXT:seminars/Resources/Private/Language/locallang_db.xlf:tx_seminars_seminars.event_type',
             'config' => [
-                'type' => 'select',
+                'type' => \OliverKlee\Seminars\BackEnd\TceForms::getSelectType(),
```
Comment (oliverklee) [inline Configuration/TCA/tx_seminars_seminars.php]: Could you please undo these changes? This method is gone from master. Thanks!

Comment (leofuehringer) [reply Configuration/TCA/tx_seminars_seminars.php]: Ok... I'll undo these changes and make an new pullrequest for the Version 2.1.1.
Where can I find this method? 
Thanks!

Comment (oliverklee) [reply Configuration/TCA/tx_seminars_seminars.php]: The method is no longer there, i.e., I've completely removed it.

**Comment by leofuehringer** [comment]
I've added a new pullrequest.
